src/card.css

```
@@ -0,0 +1,90 @@
+.card {
+  padding-top: 32px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px;
+  align-items: center;
+}
+
+.card__stars-review {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: 0 19px;
+}
+
+.card__description {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: left;
+  margin: 40px 16px 0;
+}
+
+.card__number {
+  display: inline-block;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  padding-left: 16px;
+  color: #616070;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px;
+}
```

```suggestion
.card__price {
  display: flex;
  justify-content: space-between;
  align-items: center;
  margin: 0 16px;
}
```
  flex-direction: row; - default position, you shouldn't write it